### ISSUE TITLE
Scroll buttons are visible to a11y tools when visibly hidden.

### ISSUE THREAD

**Issue by bicknellr** [PolymerElements/paper-tabs#216]
### Description
Screen readers are able to navigate to the scroll buttons when they are not visible.

### Expected outcome
Screen readers are *not* able to navigate to the scroll buttons when they are not visible.

### Actual outcome
Screen readers are able to navigate to the scroll buttons when they are not visible.

### Live Demo
https://www.webcomponents.org/element/PolymerElements/paper-tabs/demo/demo/index.html

### Steps to reproduce
Using a screen reader, navigate to any set of scrollable tabs within the demo. The first and last focusable areas within the tabs will always be the scroll buttons, regardless of their visibility.

### Browsers Affected
(all)
